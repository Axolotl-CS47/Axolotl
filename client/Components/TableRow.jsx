import React from 'react';
const TableRow = (props) => {
  console.log("props", props);
return (
  <tr>
  <td>{props.rowData.departDateTime}</td>
  <td>{props.rowData.arrivalDateTime}</td>
  <td>{props.rowData.destAirport}</td>
  <td>{props.rowData.duration}</td>
  <td>{props.rowData.flightNumber}</td>
  </tr>
);
}
export default TableRow;