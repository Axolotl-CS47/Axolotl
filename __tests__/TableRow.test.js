/**
 * @jest-environment jsdom
 */

 import React from 'react';
 import '@testing-library/jest-dom'
 import renderer from 'react-test-renderer';
import TableRow from '../client/components/TableRow';
 
 it('renders TableRow', async () => {
  const component = renderer.create(
    <TableRow rowData={<td>Country</td>}/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
 });
 
