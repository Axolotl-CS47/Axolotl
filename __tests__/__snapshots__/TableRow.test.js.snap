// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TableRow 1`] = `
<tr>
  <td />
  <td />
  <td />
  <td />
  <td />
</tr>
`;
